src/sentry/api/endpoints/project_group_index.py

```diff
@@ -77,6 +78,7 @@ def get(self, request: Request, project) -> Response:
                                    ``"is:unresolved"`` is assumed.)
         :qparam string environment: this restricts the issues to ones containing
                                     events from this environment
+        :qparam list hash: hashes of groups to return, overrides every other query parameter, only returning list of groups found from hashes
```

Comment:
In this case, should we return an error if a user tries to use this in combination with any other params just to make sure they don't accidentally hit this without realizing it?